### ISSUE TITLE
byte-string with an odd number of characters not recognized 

### ISSUE THREAD

**Issue by juszczakn** [greghendershott/racket-mode#366]
I imagine this is very similar to issue #362.

`(define a #"asd")` is not recognized correctly while

`(define a #"asdf")` is.

**Comment by greghendershott** [comment]
Oh derp. I needed a non-lazy `zero-or-more`.

Thank you for noticing and letting me know!

**Comment by greghendershott** [comment]
I probably shouldn't do things before finishing my first cup of coffee.

Good news is, commit 4c1f709 fixes this and is merged to `master`.

Bad news is, I referred to issue #363 in the commit message, example.rkt regression test, and even named the short-lived topic branch `issue-363`. So, I'll go reopen #363 which got auto-closed. And I'm manually closing this. And I guess I should fix the regression test comment to mention this #366 not #363.

TL;DR derp with a whipped derp sauce on top.
